lib/whitelistedNpmAdvisories.json

```diff
@@ -0,0 +1,3 @@
+{
+  "advisories": [ 1523 ]
```

Comment:
could you leave a comment explaining why it's exempt and when this advisory can be removed? thanks